modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase

```diff
@@ -2849,6 +2849,62 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-11392. Can see timestamps from payment transaction history"
+	@priority = 3
+	test CanViewOrderTransactionHistoryTimestamps {
+		property portal.acceptance = "true";
+
+		task ("Given a Minium site with a buyer user") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+			JSONAccountEntry.addAccountEntry(
+				accountEntryName = "Commerce Account",
+				accountEntryType = "Business");
+		}
+
+		task ("And a user checks out an order ") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "test@liferay.com");
+
+			FrontStore.addInitializerProductToCart(
+				productName = "U-Joint",
+				productQuantity = 1);
+
+			CommerceCheckout.gotoCheckoutViaMiniCart();
+		}
+
+		CommerceCheckout.initializersCheckout(
```

Comment:
Is this outside of the task? i think it should go inside the curly braces of the task